index.html

```diff
@@ -362,32 +367,26 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 "PERFORMANCE-TIMELINE-2#the-performanceentry-interface">PerformanceEntry</dfn></code>
 interface:</p>
 <dl data-link-for="PerformanceResourceTiming">
-<dt><dfn>name</dfn></dt>
-<dd>This attribute MUST return the <a data-cite=
-"HTML#resolve-a-url">resolved URL</a> of the requested resource.
-This attribute MUST NOT change even if the <a data-cite=
-"FETCH#concept-fetch">fetch</a> redirected to a different URL.</dd>
+<dt><dfn>name</dfn>
+<dd>The <a>name</a> getter steps are to return the
+<a data-for="PerformanceResourceTiming">requested URL</a>
 <dt><dfn>entryType</dfn></dt>
-<dd>The <a>entryType</a> attribute MUST return the DOMString
-"<code>resource</code>".</dd>
+<dd>The <a>entryType</a> getter steps are to return the DOMString "<code>resource</code>".</dd>
 <dt><dfn>startTime</dfn></dt>
-<dd>The <a>startTime</a> attribute MUST return a {{DOMHighResTimeStamp}}
-[[HR-TIME-2]] with the time immediately before the user agent
-starts to queue the resource for <a data-cite="FETCH#concept-fetch"
-data-lt='fetch'>fetching</a>. If there are HTTP redirects or
-<a data-cite="HTML#or-equivalent">equivalent</a> when fetching the
-resource, and if the <a>timing allow check</a> algorithm passes, this
-attribute MUST return the same value as <a>redirectStart</a>.
-Otherwise, this attribute MUST return the same value as
-<a>fetchStart</a>.</dd>
+<dd><p data-dfn-for="PerformanceResourceTiming">The <a>startTime</a> getter
+steps are to <a>convert fetch timestamp</a> for <a>this</a>'
```

Comment:
nit: sadly it would be `this's`, here and throughout.

Reply:
fixed